pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -6061,6 +6136,20 @@ def plot_traces(self, prob_label, data_2d, qbn):
                 'legend_ncol': legend_ncol,
                 'legend_bbox_to_anchor': legend_bbox_to_anchor,
                 'legend_pos': legend_pos}
+            if self.proc_data_dict.get('percent_data_after_presel',
+                                       False):
+                textstr = "Preselection {} = \n {}".format(qbn,
+                    self.proc_data_dict.get('percent_data_after_presel')[qbn])
+                self.plot_dicts['text_msg_{}_{}_{}'.format(
+                row, qbn, prob_label)] = {
+                    'fig_id': figure_name,
+                    'ypos': -0.2,
+                    'xpos': -0.1,
+                    'horizontalalignment': 'left',
+                    'verticalalignment': 'top',
+                    'box_props': None,
+                    'plotfn': self.plot_text,
+                    'text_string': textstr}
```

Comment:
I would suggest to wrap this and the textstring addition in lines 6391-6392 below into a method of the base MultiQubit_TimeDomain_Analysis, because it will in principle need to be added to several classes. 
I would already add a call to this method in prepare_projected_data_plot of the base MultiQubit_TimeDomain_Analysis. It makes sense that it should be there. Lots of children use the figure prepared in that base method and just add to it (Ex Rabi, Ramsey, T1). But then having it as a method will also make it easy to add it to other classes which do not do this (like the MultiCZgate_Calib_Analysis or QScale)